### PR TITLE
Upleveled Contribution guidelines, edited text.

### PR DESCRIPTION
**Pull request by MCamp859**
Reorganized text and edited per instructions in Issue #828.
Signed-off-by: MCamp859 <maryx.camp@intel.com>

### CODE REVIEW

**Comment by MCamp859** [comment]
@biapalmeiro  please review this PR.

File: source/collaboration/collaboration.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. _collaboration:
+
+How to contribute
```
Comment (mvincerx) [inline source/collaboration/collaboration.rst]: Revise doc title to "Contribute".

**Comment by mvincerx** [comment]
@krisdale, Do you have feedback on this?

**Comment by biapalmeiro** [comment]
@MCamp859 It looks great. Thanks everyone.

File: source/collaboration/collaboration.rst (hunk 1 above)
Comment (mvincerx) [reply source/collaboration/collaboration.rst]: @MCamp859 Could you please change the title of doc, as above?

**Comment by mvincerx** [comment]
LGTM.  Pushing this through after discussion in standup this morning. Thanks all for feedback. 
Closes GH-828

**Pull request merged**